src/ott/initializers/linear/initializers.py

```diff
@@ -313,3 +338,80 @@ def body_fn(i: int, f: jnp.ndarray) -> jnp.ndarray:
     return f.at[i].set(new_f)
 
   return jax.lax.fori_loop(0, len(f), body_fn, f)
+
+
+@jax.tree_util.register_pytree_node_class
+class SubsampleInitializer(DefaultInitializer):
+  """Subsample initializer :cite:`thornton2022rethinking:22`.
+
+  Subsample each :class:~ott.geometry.point_cloud.PointCloud, then compute closed from
+  Sinkhorn potential between subsampled approximations. Use this potential
```

Comment:
Pet peeve: would not start another sentence, would use, e.g., `... approximations and use this ...`

Reply:
sure